src/sentry/static/sentry/app/components/search/sources/apiSource.jsx

```diff
@@ -120,6 +127,42 @@ async function createIntegrationResults(integrationsPromise, orgId) {
   );
 }
 
+async function createSentryAppResults(sentryAppPromise, orgId) {
+  const sentryApps = (await sentryAppPromise) || [];
```

Comment:
When/why would `sentryAppPromise` return a falsey value?

Reply:
@billyvg It only it only returns `undefined` in a test, even though the function is mocked. Not sure why it does this. We do this for other APIs that wouldn't return falsy values normally either. It's not ideal, but I think it's probably to just be consistent.